dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/FileVerificationService.kt

```diff
@@ -8,40 +8,40 @@ import org.springframework.web.multipart.MultipartFile
 import java.io.IOException
 
 /**
- * A service for performing basic PDF sanity checks
+ * A service for performing basic sanity checks on files uploaded by user
  */
 @Component
-class PdfVerificationService {
+class FileVerificationService {
     private val logger = LoggerFactory.getLogger(javaClass)
 
     /**
-     * A function that performs surface-level checks to ensure that an uploaded document is indeed a PDF.
+     * A function that performs surface-level checks to ensure that an uploaded file is indeed a PDF.
      * This function does not enforce anything else.
      * In particular a file passing this function is in no way guaranteed to be safe to open.
      */
-    fun assertThatDocumentLooksLikeAPdf(document: MultipartFile, correlationId: String) {
+    fun assertThatFileLooksLikeAPdf(file: MultipartFile, correlationId: String) {
         try {
-            checkIfPdfDocumentIsEmpty(document.bytes, correlationId)
+            checkIfPotentialPdfFileIsEmpty(file.bytes, correlationId)
```

Comment:
I would completely detach anything here from the pdf format. We can upload anything right now. So why not just `checkIfFileIsEmpty`.

Reply:
Because the emptiness-check only works, if the file can be interpreted as pdf.
But because the file can also be something else, it checks if the "potential" pdf is empty.

Reply:
Aaaaah. I see. It is not just a check for the file size.